charts/partials/templates/_proxy-init.tpl

```diff
@@ -50,10 +50,13 @@ securityContext:
   runAsNonRoot: false
   runAsUser: 0
 terminationMessagePolicy: FallbackToLogsOnError
-{{- if .Values.global.proxyInit.saMountPath }}
 volumeMounts:
+- mountPath: {{.Values.global.proxyInit.xtMountPath.mountPath}}
+  name: {{.Values.global.proxyInit.xtMountPath.name}}
+  subPath: {{.Values.global.proxyInit.xtMountPath.subPath}}
```

Comment:
Good catch @alpeb I added those statements.

The reason I used `subPath` is to limit the volume to just `/run/xtables.lock` rather than the entire `/run` directory. I may be misusing `subPath` and will change it if that's not the right way to limit the scope of the volume

Reply:
Right, in that case I believe the right format would be
``` yaml
mountPath: /run/xtables.lock
name: linkerd-proxy-init-xtables-lock
subPath: xtables.lock
```
but even so, it doesn't feel very kosher to me given `xtables.lock` doesn't even exist when this gets mounted. IMO we should leave things simpler and omit `subPath`.